spec/requests/users/registrations_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Registrations", type: :request do
+  it "should register new user" do
+    user_payload = {
+      name: "name",
+      surname: "surname",
+      email: "test@example.com",
+      password: "password",
+      password_confirmation: "password"
+    }
+
+    post "/user", params: {
+      user: user_payload
+    }
```

Comment:
You can add an assertion here

```suggestion
    expect { post "/user", params: { user: user_payload } }
      .to change(User, :count).by(1)
```